kDriveCore/Services/AppContextService.swift

```diff
@@ -23,6 +23,9 @@ public enum DriveAppContext: String {
     /// Current execution context is the main app
     case app
 
+    /// Current execution context is testing
+    case appTests
```

Comment:
This replaces the old `TEST` flag that was broken.

As set at launch time, rather than dynamically computed, this is relatively cost free and leverages the existing abstraction that does this job already.